pkg/kubelet/pod/pod_manager.go

```diff
@@ -77,10 +78,14 @@ type Manager interface {
 	// this means deleting the mappings related to mirror pods.  For non-
 	// mirror pods, this means deleting from indexes for all non-mirror pods.
 	DeletePod(pod *v1.Pod)
-	// DeleteOrphanedMirrorPods deletes all mirror pods which do not have
-	// associated static pods. This method sends deletion requests to the API
-	// server, but does NOT modify the internal pod storage in basicManager.
-	DeleteOrphanedMirrorPods()
+	// GetOrphanedMirrorPodNames returns names of orphaned mirror pods
+	GetOrphanedMirrorPodNames() []string
+	// GetPendingPodUID returns UID for static pods pending termination
+	GetPendingPodUID(podFullName string) (kubetypes.UIDWithGracePeriod, bool)
```

Comment:
the implementation is not specific to mirror pods.

the backing map persists all pods mirror or not when i look at DeletePod

Reply:
i think a more descriptive name is useful.  its not really pending pods.
its pods undergoing graceful termination.
if we are changing the interface, we should have clearer names as pending has meaning in other parts of system.

Reply:
Renamed to GetUIDForPodPendingTermination

Thanks for the suggestion.